curriculum/challenges/english/blocks/workshop-shape-manager/69145bb447c0c14bf43bff1f.md

```diff
@@ -0,0 +1,422 @@
+---
+id: 69145bb447c0c14bf43bff1f
+title: Step 18
+challengeType: 25
+dashedName: step-18
+---
+
+# --description--
+
+To get the function to calculate the results, you can use an `if...else if` or `switch` statement.
```

Comment:
I think this could be a bit too much for a single step. And the `never` type is not introduced.